mypy/test/testtypes.py

```diff
@@ -225,75 +225,75 @@ def assert_erase(self, orig: Type, result: Type) -> None:
 
     def test_is_more_precise(self) -> None:
         fx = self.fx
-        assert_true(is_more_precise(fx.b, fx.a))
-        assert_true(is_more_precise(fx.b, fx.b))
-        assert_true(is_more_precise(fx.b, fx.b))
-        assert_true(is_more_precise(fx.b, fx.anyt))
-        assert_true(is_more_precise(self.tuple(fx.b, fx.a),
+        assert is_more_precise(fx.b, fx.a)
+        assert is_more_precise(fx.b, fx.b)
+        assert is_more_precise(fx.b, fx.b)
+        assert is_more_precise(fx.b, fx.anyt)
+        assert is_more_precise(self.tuple(fx.b, fx.a,
```

Comment:
Looks like the find and replace wasn't quite right

Reply:
😅